build.sbt

```diff
@@ -25,26 +25,26 @@ libraryDependencies ++= Seq(
   "org.eclipse.jgit"                %  "org.eclipse.jgit.archive"     % "4.7.0.201704051617-r",
   "org.scalatra"                    %% "scalatra"                     % ScalatraVersion,
   "org.scalatra"                    %% "scalatra-json"                % ScalatraVersion,
-  "org.json4s"                      %% "json4s-jackson"               % "3.5.0",
+  "org.json4s"                      %% "json4s-jackson"               % "3.5.1",
   "io.github.gitbucket"             %% "scalatra-forms"               % "1.1.0",
-  "commons-io"                      %  "commons-io"                   % "2.4",
-  "io.github.gitbucket"             %  "solidbase"                    % "1.0.0",
+  "commons-io"                      %  "commons-io"                   % "2.5",
+  "io.github.gitbucket"             %  "solidbase"                    % "1.0.2",
   "io.github.gitbucket"             %  "markedj"                      % "1.0.10",
-  "org.apache.commons"              %  "commons-compress"             % "1.11",
+  "org.apache.commons"              %  "commons-compress"             % "1.13",
   "org.apache.commons"              %  "commons-email"                % "1.4",
-  "org.apache.httpcomponents"       %  "httpclient"                   % "4.5.1",
-  "org.apache.sshd"                 %  "apache-sshd"                  % "1.2.0",
-  "org.apache.tika"                 %  "tika-core"                    % "1.13",
+  "org.apache.httpcomponents"       %  "httpclient"                   % "4.5.3",
+  "org.apache.sshd"                 %  "apache-sshd"                  % "1.4.0",
+  "org.apache.tika"                 %  "tika-core"                    % "1.14",
   "com.github.takezoe"              %% "blocking-slick-32"            % "0.0.8",
-  "joda-time"                       %  "joda-time"                    % "2.9.6",
-  "com.novell.ldap"                 %  "jldap"                        % "2009-10-07",
-  "com.h2database"                  %  "h2"                           % "1.4.192",
-  "mysql"                           %  "mysql-connector-java"         % "5.1.39",
-  "org.postgresql"                  %  "postgresql"                   % "9.4.1208",
-  "ch.qos.logback"                  %  "logback-classic"              % "1.1.7",
-  "com.zaxxer"                      %  "HikariCP"                     % "2.4.6",
-  "com.typesafe"                    %  "config"                       % "1.3.0",
-  "com.typesafe.akka"               %% "akka-actor"                   % "2.4.12",
+  "joda-time"                       %  "joda-time"                    % "2.9.9",
+  "com.novell.ldap"                 %  "jldap"                        % "4.3",
```

Comment:
> 4.3 is older version?

According to VersionEye :) (what I used)
https://www.versioneye.com/user/projects/590a19855420bc001227f58b

The numbering doesn't seems OK however, since from the sources the most recent version should be 4.6  http://www.openldap.org/devel/gitweb.cgi?p=openldap-jldap.git;a=commit;h=bae9f556e97ad5bb0f76fe897c9b298a6021e244